mne/time_frequency/psd.py

```diff
@@ -168,7 +168,7 @@ def compute_epochs_psd(epochs, picks=None, fmin=0, fmax=np.inf, tmin=None,
     else:
         time_mask = Ellipsis
 
-    data = epochs.get_data()[:, picks][..., time_mask]
+    data = epochs.get_data()[:, picks][:, :, time_mask]
```

Comment:
Is this really the only way to do this? This patch applies to the development version too. Changing two lines of code shouldn't be this complicated...